visidata/features/describe.py

```diff
@@ -94,17 +94,16 @@ def reloadColumn(self, srccol):
                 except Exception as e:
                     d['errors'].append(sr)
 
-            d['mode'] = self.calcStatistic(d, mode, vals)
+            d['mode'] = self.calcStatistic(mode, vals)
             if vd.isNumeric(srccol):
                 for func in [min, max, sum, median]:  # use type
-                    d[func.__name__] = self.calcStatistic(d, func, vals)
+                    d[func.__name__] = self.calcStatistic(func, vals)
                 for aggrname in vd.options.describe_aggrs.split():
-                    func = vd.getGlobals()[aggrname]
-                    d[func.__name__] = self.calcStatistic(d, func, vals)
+                    aggr = vd.aggregators_vals[aggrname]
```

Comment:
Can't we just use `vd.aggregators[aggrname].func`, instead of maintaining a separate dict of funcs?

Reply:
They have different function signatures! One is (srcCol, rows), the other (values). I added an optional funcValues argument!